lib/sass/calcite-web.scss

```diff
@@ -30,17 +30,17 @@ $include-sticky:                 $generate-css    !default;
 
 // Type Output
 $include-type:                   $generate-css    !default;
-  $include-primary-family:       true             !default;
```

Comment:
It was not clear to me how the names `$include-type` and `$include-primary-family` revealed anything relational or subgrouping. Is this due to legacy variable names? Should the sub-grouping be evident when finding these variables in the wild?